packages/valory/skills/contribution_abci/skill.yaml

```diff
@@ -111,6 +111,11 @@ models:
       round_timeout_seconds: 30.0
       service_id: contribution_service
       service_registry_address: null
+      setup:
+        safe_contract_address:
+        - '0x8001bdCf80F8Fb61CdcDA48419A30b430B385ca1'
```

Comment:
The safe is not required by this service, but for the framework is mandatory so I just set it to the same as the autonomous fund.